docs/organizations/managing-security-and-risk.md

```diff
@@ -91,7 +93,7 @@ To access the findings page, access the [overview page](#dashboard) and click th
 
 ![Security and risk management findings page](images/security-risk-management-findings.png)
```

Comment:
Should these screenshots be updated with ones that already include the Segments filter? lines `94` and `100`.